### DISCUSSION TITLE
Windows 11 SMB Mount - Organization Security Policies block unauthenticated guess access

### DISCUSSION THREAD

**Discussion by hkhiro** [IceWhaleTech/CasaOS#2053] [Q&A]
getting this error - is it because there is no SMB password setup on CasaOS? Recommended way of fixing?
![image](https://github.com/user-attachments/assets/c5b53d77-5f09-48dc-b3af-ccc0bd7dfd27)

**Comment by tachikoma1373** [comment]
same issue on clean install - given everyhting i've seen about casaos i was keen but its been a right royal pain in the but so im back to of all things OMV

**Reply by hkhiro** [reply]
I did get mine to work somehow following various instructions etc and updating config files. People seem to install cockpit and portainer to top up this app. Also there is ZimaOS being developed

i used OMV for awhile and believe it or not people also running casaos alongside this. The docker plugin is optional as such in OMV

**Reply by tachikoma1373** [reply]
I tried zimaos as well and that had the same issue - which strikes me as a fairly fundamental one (especially given its apparent commonality and it seems its been known about for a while).  I appreciate the focus is on compatability with the zima hardware but given how much feedback seems to be driven by non-zima based users it was a little disappointing.  OMV just isnt as friendly and I sure i will revisit casaos/zimaos in the future but for now I'll just keep an eye on them.
